src/Compilers/CSharp/Portable/Symbols/Metadata/PE/MetadataDecoder.cs

```diff
@@ -515,8 +515,8 @@ internal override Symbol GetSymbolForMemberRef(MemberReferenceHandle memberRef,
                 HashSet<DiagnosticInfo> useSiteDiagnostics = null;
                 if (!TypeSymbol.Equals(scope, targetTypeSymbol, TypeCompareKind.ConsiderEverything2) &&
```

Comment:
>TypeCompareKind.ConsiderEverything2 [](start = 64, length = 35)

while you're here, consider fixing this TypeCompareKind

Reply:
> while you're here, consider fixing this TypeCompareKind

Fixing often requires thinking if the kind should be changed and adding tests proving one or the other. I prefer to not include this work into the current PR.

---
In reply to: [244437039](https://github.com/dotnet/roslyn/pull/32041#discussion_r244437039) [](ancestors = 244437039)